### DISCUSSION TITLE
PR merged only when all reviewers have a positive review.

### DISCUSSION THREAD

**Discussion by ogeorget-psee** [Mergifyio/mergify#5064] [Q&A]
Hi Mergifiers,

In your documentation, you provide an example to trigger actions on a PR, when all requested reviews are approved (cf. [All Requested Reviews to Be Approved](https://docs.mergify.com/examples/#id14)). 

This is a little misleading, as only 1 reviewer is actually required to approve the PR (as per `#approved-reviews-by>=1`). Reviewer leaving a comment would be treated as "approving" the PR (because of the `#review-requested=0`) unless they actually request changes. 

Is there a way to have a rule, that is triggered only when all reviewers have actually approved the PR (with a ✔️ ) ?  
I couldn't find a way to get the list of "solicited" reviewers on a PR. So I can write something like `#approved-reviews-by=#solicited-reviewers` 

Thanks 🙏🏻

**Comment by jd** [comment]
Unfortunately, GitHub makes this impossible. As you noticed, a comment is enough to consider the review "done."

**Reply by ogeorget-psee** [reply]
After a quick look at Github's API, it seems to be not possible by design indeed 😭 

It's a real shame such basic information isn't available in the API. The only feature request I could found on Github side is - https://github.com/orgs/community/discussions/39349. 

Is this me not understanding how the Pull Request system works, or people have actually no interest in the info to enable auto-merging triggering ?

**Reply by jd** [reply]
A lot of teams are frustrated by this indeed and find their workaround or trade-off (such as just using a number of approvals).

**Reply by ogeorget-psee** [reply]
We will use a label as an opt-in mechanism, even though this defeats the purpose of automating the review process. 
Thanks for the help anyway.
